### ISSUE TITLE
Result cannot be seen in dark themes

### ISSUE THREAD

**Issue by kupiqu** [dschopf/qalculate#1]
Nice work, by the way

**Comment by kupiqu** [comment]
To fix add color: theme.textColor property to lResult in FullRepresentation.qml file:

```
Label {
      id: lResult
      text: "Result"
      color: theme.textColor
      visible: false
      anchors.horizontalCenter: parent.horizontalCenter
      font.bold: true
      font.pixelSize: 40
    }
```

**Comment by dschopf** [comment]
Fixed in  192eeac
